### DISCUSSION TITLE
Coding Assistant Model

### DISCUSSION THREAD

**Discussion by aezendc** [SciSharp/LLamaSharp#893] [Q&A]
Hello everyone,

I'm wondering if there's a GGUF model specifically designed for LLAMA that supports C# or .NET development. Could you please provide any information or resources related to this?

Thank you!

**Comment by n-ate** [comment]
I've used over 50 models from hugging face already built as GGUF. 2 of them failed to load properly. 1 of them seemed rather garbage. The rest are of varying degrees of usefulness. I run 2-9B parameter sized models between Q4 K_M and Q8 K_M.
